package.json

```diff
@@ -30,12 +30,12 @@
     "node-fetch": "^2.6.0",
     "pm2": "^4.4.0",
     "pm2-promise": "^2.0.1",
-    "rainbow-bridge-lib": "near/rainbow-bridge-lib#8e5567c0277d13430f952a19bb5e16ceb0b4392c",
+    "rainbow-bridge-lib": "near/rainbow-bridge-lib#373e9f787e51703f186e83e8036dc35131742c1b",
     "rainbow-bridge-rs": "1.0.2",
     "rainbow-bridge-sol": "1.1.0",
     "request": "^2.88.2",
     "web3": "=1.2.6",
-    "rainbow-token-connector": "near/rainbow-token-connector#620ad3e45f9f722d77f88a7a39b252d5a4ead9ef"
+    "rainbow-token-connector": "near/rainbow-token-connector#0f874679dc534bfe9cbe8092584a5594a299fda1"
```

Comment:
Also add this to ci/patch_contract_version.js? (btw the name seems more proper to be patch_rainbow_deps.js or similar)

Reply:
Done